tests/data/raster/README.md

```diff
@@ -0,0 +1,6 @@
+Data Sources
+============
+
+#### `aerial`
+
+[Auckland 0.5m Rural Aerial Photos (2010-2012)](https://data.linz.govt.nz/layer/51769-auckland-05m-rural-aerial-photos-2010-2012/); Land Information New Zealand; CC-BY
```

Comment:
I'd note that you've heavily downsampled it since the title of the layer implies 0.5m/px res which is obviously inaccurate

Reply:
Done